docs/admin/authentication.md

```diff
@@ -212,6 +219,17 @@ other claims, such as `email`, depending on their provider.
 * `--oidc-groups-claim` JWT claim to use as the user's group. If the claim is present
 it must be an array of strings.
 
+Kubernetes does not provide an OpenID Connect Identity Provider.
+You can use an existing public OpenID Connect Identity Provider (such as Google, or [others](http://connect2id.com/products/nimbus-oauth-openid-connect-sdk/openid-connect-providers)).
+Or, you can run your own Identity Provider, such as CoreOS [dex](https://github.com/coreos/dex/blob/master/Documentation/getting-started.md), [Keycloak](https://github.com/keycloak/keycloak) or CloudFoundary [UAA](https://github.com/cloudfoundry/uaa).
```

Comment:
Mind linking to the dex repo (https://github.com/coreos/dex) rather than a specific document? We're going through a large rewrite and that link will be more stable.